samples/sqltransport/multi-instance/Client/App.config

```diff
@@ -0,0 +1,50 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <configSections>
+    <section name="MessageForwardingInCaseOfFaultConfig" type="NServiceBus.Config.MessageForwardingInCaseOfFaultConfig, NServiceBus.Core" />
+    <section name="UnicastBusConfig" type="NServiceBus.Config.UnicastBusConfig, NServiceBus.Core" />
+    <section name="AuditConfig" type="NServiceBus.Config.AuditConfig, NServiceBus.Core" />
+  </configSections>
+  <startup>
```

Comment:
Can you explain why we remove auto-generated sections? How is that going to help the customer understand the sample?